<ui version="4.0" >
 <class>BoundaryConditionsTurbulenceInletForm</class>
 <widget class="QWidget" name="BoundaryConditionsTurbulenceInletForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxTurbulence" >
     <property name="title" >
      <string>Turbulence</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="_2" >
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <layout class="QGridLayout" name="_3" >
          <item row="0" column="0" >
           <widget class="QComboBox" name="comboBoxTurbulence" />
          </item>
          <item row="1" column="0" >
           <widget class="QFrame" name="frameTurbDiameter" >
            <property name="minimumSize" >
             <size>
              <width>16</width>
              <height>42</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="_4" >
             <item row="0" column="0" >
              <layout class="QGridLayout" name="_5" >
               <item row="0" column="0" >
                <widget class="QLabel" name="labelDiameter" >
                 <property name="text" >
                  <string>Hydraulic diameter</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" >
                <widget class="QLineEdit" name="lineEditDiameter" >
                 <property name="minimumSize" >
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Inlet hydraulic diameter = 
4S/P with S=surface and P=perimeter of wet wall</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2" >
                <widget class="QLabel" name="labelUnitDiameter" >
                 <property name="text" >
                  <string>m</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QFrame" name="frameTurbIntensity" >
            <property name="minimumSize" >
             <size>
              <width>16</width>
              <height>70</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="_6" >
             <item row="0" column="0" >
              <layout class="QGridLayout" name="_7" >
               <item row="0" column="0" >
                <widget class="QLabel" name="labelIntensity" >
                 <property name="text" >
                  <string>Intensity</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" >
                <widget class="QLineEdit" name="lineEditIntensity" >
                 <property name="minimumSize" >
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>turbulent intensity expressed in percent </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2" >
                <widget class="QLabel" name="labelUnitIntensity" >
                 <property name="text" >
                  <string>%</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" >
                <widget class="QLabel" name="labelDiameterIntens" >
                 <property name="text" >
                  <string>Hydraulic diameter</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" >
                <widget class="QLineEdit" name="lineEditDiameterIntens" >
                 <property name="minimumSize" >
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Inlet hydraulic diameter = 
4S/P with S=surface and P=perimeter of wet wall</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2" >
                <widget class="QLabel" name="labelUnitDiameterintens" >
                 <property name="text" >
                  <string>m</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
